lib/reline/line_editor.rb

```diff
@@ -1101,35 +1101,32 @@ def input_key(key)
     @first_char = false
     completion_occurs = false
     if @config.editing_mode_is?(:emacs, :vi_insert) and key.char == "\C-i".ord
-      unless @config.disable_completion
-        result = call_completion_proc
-        if result.is_a?(Array)
-          completion_occurs = true
-          process_insert
-          if @config.autocompletion
-            move_completed_list(result, :down)
-          else
-            complete(result)
+      if !@config.disable_completion
+        process_insert(force: true)
```

Comment:
If you type `"1."` and then paste `"0.\t\t"` to the terminal, `"0."` is inserted to `@continuous_insertion_buffer` (because `in_pasting?` is true).
Without this `process_insert(force: true)`, `0.` part will be ignored.
```
irb(main):001> 1.floor
               1.abs
               1.floor[selected]
               1.ceil
```

We need to force insert `"0."` into the buffer before calculating completion list.
```
irb(main):001> 1.0.ceil
               1.0.floor
               1.0.ceil[selected]
               1.0.round
```

Reply:
Thank you for your information!
But I could not paste "0./t/t" in my environment. Can you paste it?

I got this:

```
irb(main):001> 1.__id__
               1.__send__[selected]
               1.equal?
```

Reply:
It was broken in the last commit...

Fixed and I added test for it